Python/Product/TestAdapter.Executor/Pytest/TestExecutorPytest.cs

```diff
@@ -0,0 +1,171 @@
+// Python Tools for Visual Studio
```

Comment:
rename and moved TestExecutor to Pytest/TestExecutorPytest.cs

also removed all the old unittest execution code that was at the bottom